docs/source/community/governance.md

```diff
@@ -0,0 +1,279 @@
+
+
+### Lazy consensus
+
+Decision making typically involves the following steps:
+*   Proposal *(via GitHub issue + GitHub PR)*
+*   Discussion *(in Discord channels and GitHub)*
+*   (optional) Maintainers voting (if there is active opposition + consensus is not reached through discussion)
+*   Decision
+
+Any community member can make a proposal for consideration by the community. To initiate a discussion about a new idea,
+they should create an issue or submit a PR implementing the idea to the issue tracker. This will prompt a review and,
+if necessary, a discussion of the idea. The goal of this review and discussion is to gain approval for the contribution.
+Since most people in the project community have a shared vision, there is often little discussion to reach consensus.
+
+In general, as long as nobody explicitly opposes a proposal or PR, it is recognized as having the support of the
+community. This is called lazy consensus - that is, those who have not stated their opinion explicitly have implicitly
+agreed to the proposal's implementation.
+
+Lazy consensus is a fundamental concept within the project. This process allows a large group of people to reach
+consensus efficiently as someone with no objections to a proposal need not spend time stating their position.
+
+For lazy consensus to be effective, it is necessary to allow at least 48 hours before assuming that there are no
+objections to the proposal. This requirement ensures that everyone is given enough time to read, digest, and respond to
+the proposal. This time period is chosen to be as inclusive as possible of all participants, regardless of their
+location and time commitments.
+
+### Voting
+
+Not all decisions can be made using lazy consensus. Issues such as those affecting the strategic direction or legal
+standing of the project must gain explicit approval in the form of a vote. Every member of the community is encouraged
+to express their opinions in all discussions and all votes. However, only project maintainers have binding votes for
+the purposes of decision making.
+
+
+### Changes to Governance
+
+We believe governance needs to adapt in order to be effective long term. This governance document itself can be extended
+or modified as our community and project grows and our needs change.
+
+A change in our governance structure should be a rare occurrence and should face sufficient scrutiny and review. To
+this end, the rules that apply to modifications to the Llama Stack Governance structure are more stringent:
+
+*   Governance changes are made through PRs to the repository.
+*   Lazy consensus applies to governance changes, but the proposed changes must be public for at least 7 days instead of 48 hours before they are accepted.
+*   If there is opposition to a change, a vote will be held by maintainers.
+*   Voting is asynchronous. All maintainers must be notified of a vote through the maintainer mailing list.
```

Comment:
I don't think we have a ML

Reply:
Yeah, we don't, we'd need to create one.

Reply:
discord